src/include/gcc.h

```diff
@@ -157,6 +176,25 @@ WT_ATOMIC_FUNC(i64, int64_t, int64_t *vp, int64_t v)
 WT_ATOMIC_FUNC(iv64, int64_t, volatile int64_t *vp, volatile int64_t v)
 WT_ATOMIC_FUNC(size, size_t, size_t *vp, size_t v)
 
+/*
+ * We can't use the WT_ATOMIC_FUNC macro for booleans as add and sub are unsupported with clang.
+ * Define them individually.
+ */
+static inline bool
+__wt_atomic_loadb(bool *vp)
```

Comment:
I'd like to keep this as is since it aligns with the other atomic functions: `__wt_atomic_load8`, `__wt_atomic_loadv64`, but perhaps we rename to `__wt_atomic_loadbool`? 

I'll make the change to untyped below since that feels distinct from the typed functions

Reply:
Well wouldn't it then be load1? Maybe loadbool is better.

Reply:
`b` could definitely be misinterpreted as byte/bool/bit.

Reply:
> Well wouldn't it then be load1? Maybe loadbool is better.
It's one bit of information but I wouldn't want to assume our compilers agree. Also I think the min size we can index is 8.

byte/bool/bit is a good point. I'll make this `__wt_atomic_loadbool`

Reply:
Renamed in [c3e7406](https://github.com/wiredtiger/wiredtiger/pull/10236/commits/c3e74065e3cd9bbd55695ff13c03a2c1247cbb1f)

Reply:
Please also change store bool.